### PR TITLE
Timeout is an option of `waitFor`, not `then`

### PR DESCRIPTION
**Pull request by Carreau**
should at least mitigeate #762

### CODE REVIEW

**Comment by Carreau** [comment]
ping @jdfreder

File: notebook/tests/util.js [hunk 1]
```diff
@@ -166,11 +166,15 @@ casper.wait_for_output = function (cell_num, out_num) {
             },
             // pass parameter from the test suite js to the browser code js
             {c : cell_num, o : out_num});
+        },
+        function then() { },
+        function timeout() {
+            this.echo("wait_for_output timed out on cell "+cell_num+", waiting for "+out_num+"outputs .");
+            pn = this.evaluate(function get_prompt(c) {
```
Comment (Carreau) [inline notebook/tests/util.js]: that's wrong.

Comment (jdfreder) [inline notebook/tests/util.js]: var pn

Comment (Carreau) [inline notebook/tests/util.js]: done.

**Comment by jdfreder** [comment]
Awesome! :cake:

**Pull request merged**